data-prepper-plugins/drop-events-processor/src/test/java/com/amazon/dataprepper/plugins/processor/drop/DropEventsProcessorTests.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.amazon.dataprepper.plugins.processor.drop;
+
+import com.amazon.dataprepper.model.configuration.PluginSetting;
+import com.amazon.dataprepper.model.event.Event;
+import com.amazon.dataprepper.model.event.JacksonEvent;
+import com.amazon.dataprepper.model.record.Record;
+import org.junit.jupiter.api.Test;
+
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+public class DropEventsProcessorTests {
+    private String messageInput;
+    private DropEventsProcessor dropProcessor;
+    private PluginSetting pluginSetting;
+    private final String PLUGIN_NAME = "drop_events";
+
+    @Test
+    void doExecute() {
```

Comment:
I'd recommend renaming your unit test function names. Unit tests usually start with the prefix "test". I also consider it bad practice to name the test only the name of the function it is testing. Consider something like "public void testGivenListOfEventsWhenExecuteThenEmptyCollectionReturned" or "public void testEmptyCollectionReturned".

testGiven...When...Then is a little verbose but a good template to help answer the question "Why am I making this test?"

Reply:
Thanks for catching that! @dapowers87 , there is some guidance here: https://github.com/opensearch-project/data-prepper/blob/main/docs/developer_guide.md?plain=1#L174